_posts/0000-01-02-Martz73.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
#They told me to  Replace line 5 of my file with a quotation or meme and witty caption, but I just didn't want to. ;)